### PR TITLE
chore: Add getting-started scripts PR template

### PR DESCRIPTION
**Pull request by Techassi**
Part of https://github.com/stackabletech/issues/pull/606

### CODE REVIEW

**Review by NickLarsenNZ — APPROVED** [review]
LGTM.

Just thinking I'm going to have to get used to using the PR templates, as currently I push the branch and click the link that Github sends back to create a PR with an empty template :thinking:

**Comment by Techassi** [comment]
Personal reminder: This still needs to be rolled-out in downstream repos.

File: template/.github/PULL_REQUEST_TEMPLATE/pre-release-getting-started-script.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Check and Update Getting Started Script
```
Comment (NickLarsenNZ) [inline template/.github/PULL_REQUEST_TEMPLATE/pre-release-getting-started-script.md:1]: I think the linter isn't running, as in the downstream repos it picks this up:

```
template/.github/PULL_REQUEST_TEMPLATE/pre-release-getting-started-script.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "## Check and Update Getting St..."]
```

File: template/.github/PULL_REQUEST_TEMPLATE/pre-release-rust-deps.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
 ## Bump Rust Dependencies for Stackable Release XX.(X)X
```
Comment (NickLarsenNZ) [inline template/.github/PULL_REQUEST_TEMPLATE/pre-release-rust-deps.md:1]: And this one:

```
template/.github/PULL_REQUEST_TEMPLATE/pre-release-rust-deps.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "## Bump Rust Dependencies for ..."]
```

**Pull request merged**